drivers/Aqara/aqara-presence-sensor/src/discovery_mdns.lua

```diff
@@ -0,0 +1,143 @@
+local log = require "log"
+local mdns = require "st.mdns"
+local net_utils = require "st.net_utils"
+
+local discovery_mdns = {}
+
+local function byte_array_to_plain_text(byte_array)
+  return string.char(table.unpack(byte_array))
```

Comment:
I am not sure why this would be needed? I thought all the responses from the mdns API would be strings